scripts/setup-emlinux

```diff
@@ -1,32 +1,62 @@
 #!/bin/bash
 
+clean_up()
+{
+    unset BUILDDIR REPOS SYNC_POKY POKY_REV
+}
+
+# save error during 'source setup-emlinux'
+_ERROR=0
+trap '_ERROR=$?' ERR
+
 BUILDDIR="build"
 if [ -n "$1" ]; then
     BUILDDIR=$1
 fi
 
-sudo apt-get update
-sudo apt-get install gawk wget git-core diffstat unzip texinfo gcc-multilib \
-     build-essential chrpath socat cpio python python3 python3-pip python3-pexpect \
-     xz-utils debianutils iputils-ping libsdl1.2-dev xterm
+REPOS=${PWD}/repos
+if [ ! -e ${REPOS}/poky ]; then
+    git clone -b master https://git.yoctoproject.org/git/poky ${REPOS}/poky
+fi
+if [ ! -e ${REPOS}/meta-debian ]; then
+    git clone -b master \
+	https://github.com/meta-debian/meta-debian.git ${REPOS}/meta-debian
+fi
+if [ ! -e ${REPOS}/meta-debian-extended ]; then
+    git clone -b master \
+	https://github.com/iwamatsu/meta-debian-extended.git \
```

Comment:
Can we change meta-debian-extended repo from Iwamatsu-san's to miraclelinux's?

Reply:
Yes. I will use miraclelinux's repository.